### DISCUSSION TITLE
argo-workflows/workflow-executors/

### DISCUSSION THREAD

**Discussion by giscus** [argoproj/argo-workflows#10053] [General]
# argo-workflows/workflow-executors/

https://argoproj.github.io/argo-workflows/workflow-executors/

**Comment by xyrobo** [comment]
Exit Code 64¶
The emissary will exit with code 64 if it fails. This may indicate a bug in the emissary.

could you figure out this? we use argo workflow in production , this error was  emit too many time

**Reply by weretheone** [reply]
This is something which affects us as well on Windows nodes.
